config/src/main/java/io/scalecube/config/MultimapConfigPropertyImpl.java

```diff
@@ -0,0 +1,73 @@
+package io.scalecube.config;
+
+import io.scalecube.config.source.LoadedConfigProperty;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class MultimapConfigPropertyImpl<T> extends AbstractSimpleConfigProperty<Map<String, List<T>>> implements MultimapConfigProperty<T> {
+
+  MultimapConfigPropertyImpl(String name,
+                             Map<String, LoadedConfigProperty> propertyMap,
+                             Map<String, Map<Class, PropertyCallback>> propertyCallbackMap,
+                             Function<String, T> valueParser) {
+    super(name, getMapPropertyClass(valueParser), propertyMap, propertyCallbackMap, toMultimapPropertyParser(valueParser));
+  }
+
+  @Override
+  public Map<String, List<T>> value(Map<String, List<T>> defaultValue) {
+    return value().orElse(defaultValue);
+  }
+
+  @Override
+  public Map<String, List<T>> valueOrThrow() {
+    return value().orElseThrow(this::newNoSuchElementException);
+  }
+
```

Comment:
This parser is great but look at class `ObjectPropertyField` (35-44), this is what enables list in object config property. We need to add support of multimap there.

Reply:
done